electron/launcher.ts

```diff
@@ -96,7 +96,8 @@ async function launch(
       return
     }
   }
-  const exe = targetExe && isLegendary ? `--override-exe ${targetExe}` : ''
+  const exe =
```

Comment:
is this possible already with `gogdl`, @imLinguin ?
There is any risk of breaking GOGDL or we can keep it?